src/dbus_fast/auth.py

```diff
@@ -68,6 +68,8 @@ def __init__(self, uid: int = None) -> None:
     def _authentication_start(self, negotiate_unix_fd: bool = False) -> str:
         self.negotiate_unix_fd = negotiate_unix_fd
         uid = self.uid
+        if uid == -1:
```

Comment:
Does this have to be `-1`?

Reply:
Nope, but it seemed reasonable to take an int that would not be valid as a uid.

Reply:
Lets make it a named constant so it can be changed later under the hood without having a breaking change